internal/nightfallconfig/nightfall_config.go

```diff
@@ -113,5 +119,12 @@ func GetNightfallConfigFile(workspacePath, fileName string, logger logger.Logger
 		nightfallConfig.MaxNumberRoutines = MaxConcurrentRoutinesCap
 	}
 	nightfallConfig.FileExclusionList = append(nightfallConfig.FileExclusionList, nightfallConfigFilename)
+	// must be one of notice, warning, or failure
+	if _, ok := annotationLevels[nightfallConfig.AnnotationLevel]; !ok {
+		nightfallConfig.AnnotationLevel = AnnotationLevelFailure
+		if nightfallConfig.AnnotationLevel != "" {
```

Comment:
Because we are setting `nightfallConfig.AnnotationLevel = AnnotationLevelFailure` wont `nightfallConfig.AnnotationLevel != ""` always be `true`? 

In the log message below we will be logging the `AnnotationLevelFailure` instead of the actual input annotation level